<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICMachineConfigPage</class>
 <widget class="QWidget" name="ICMachineConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Tolerance</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="toleranceLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X1 Acceleration and Deceleration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="xADEdit"/>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>X1 Max Speed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="xMaxSpeedEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y1 Acceleration and Deceleration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="yADEdit"/>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Y1 Max Speed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="yMaxSpeedEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Z Acceleration and Deceleration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="zADEdit"/>
   </item>
   <item row="3" column="5">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Z Max Speed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="zMaxSpeedEdit"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>X2 Acceleration and Deceleration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="x2ADEdit"/>
   </item>
   <item row="4" column="5">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>X2 Max Speed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="x2MaxSpeedEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Y2 Acceleration and Deceleration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="y2ADEdit"/>
   </item>
   <item row="5" column="5">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Y2 Max Speed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="y2MaxSpeedEdit"/>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>A Acceleration and Deceleration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="aADEdit"/>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>A Max Speed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="aMaxSpeedEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>B Acceleration and Deceleration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="bADEdit"/>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>B Max Speed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="bMaxSpeedEdit"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>C Acceleration and Deceleration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="cADEdit"/>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="label_25">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string>C Max Speed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="6">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="cMaxSpeedEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
